charts/jaeger/Chart.yaml

```diff
@@ -34,7 +34,7 @@ dependencies:
     repository: https://helm.elastic.co
     condition: provisionDataStore.elasticsearch
   - name: kafka
-    version: ^19.1.5
+    version: ^18.0.8
```

Comment:
Why was this reverted to 18.x?

Reply:
@gburton1  If i have to guess it would have been an oversight and not intentional change. Maybe during squashing of the commits/resolving conflicts in this MR, this change got in.

Reply:
Hi @gburton1, it was not intentional. It probably happened during the squash. Do you want to create a new PR to revert this change?